api/producer/createDocumentReference/src/v1/handler.py

```diff
@@ -33,16 +39,15 @@
     DocumentReference as StrictDocumentReference,
 )
 
-from api.producer.createDocumentReference.src.constants import PersistentDependencies
-from api.producer.createDocumentReference.src.v1.constants import API_VERSION
+log_action = make_common_log_action()
 
 
 class LogReference(Enum):
-    CREATE001 = "Parsing request body"
-    CREATE002 = "Determining whether document reference will supersede"
-    CREATE003 = "Validating producer permissions"
-    CREATE004 = "Determining whether document reference will supersede"
-    CREATE005 = "Saving document pointer to db"
+    CREATE_REQUEST = "Parsing request body"
+    CREATE_SUPERSEDE_CHECK = "Determining whether document reference will supersede"
+    CREATE_SUPERSEDING = "Mark the document for superseding"
+    CREATE_PERMISSIONS = "Validating producer permissions"
+    CREATE_DB = "Saving document pointer to db"
```

Comment:
Very minor and can be ignored but a name suggestion of CREATE_SAVE_TO_DB or something more explicit and is it could be misinterpreted that this is creating a DB - but for the most part I doubt they would ever make that mistake 😅